### ISSUE TITLE
With the model training, the training loss becomes nan.

### ISSUE THREAD

**Issue by challengesll** [biolib/openprotein#56]
Are there any requirements for the loaded training data? I use proteinNet's txt format file to train with more than 100 batches, then the training loss becomes nan.  Has anyone encountered a similar problem？
